site/content/how-to/traffic-management/routing-traffic-to-your-app.md

```diff
@@ -132,7 +132,7 @@ spec:
 EOF
 ```
 
-This gateway is associated with the NGINX Gateway Fabric through the **gatewayClassName** field. The default installation of NGINX Gateway Fabric creates a GatewayClass with the name **nginx**. NGINX Gateway Fabric will only configure gateways with a **gatewayClassName** of **nginx** unless you change the name via the `--gatewayclass` [command-line flag](/docs/cli-help.md#static-mode).
+This gateway is associated with the NGINX Gateway Fabric through the **gatewayClassName** field. The default installation of NGINX Gateway Fabric creates a GatewayClass with the name **nginx**. NGINX Gateway Fabric will only configure gateways with a **gatewayClassName** of **nginx** unless you change the name via the `--gatewayclass` [command-line flag]({{< relref "/reference/cli-help#static-mode" >}}).
```

Comment:
```suggestion
This gateway is associated with the NGINX Gateway Fabric through the **gatewayClassName** field. The default installation of NGINX Gateway Fabric creates a GatewayClass with the name **nginx**. NGINX Gateway Fabric will only configure gateways with a **gatewayClassName** of **nginx** unless you change the name via the `--gatewayclass` [command-line flag]({{< relref "/reference/cli-help.md#static-mode" >}}).
```

If you're referencing a page anchor, you need to add the markdown extension when using a Hugo reference.

This is a convention I don't remember often, so I'm working on always adding `.md` habitually for safety.

Reply:
While it's possible to make relrefs work without the .md extension, I tend to always add it. It is required for anchors and it will be helpful when migrating from Hugo to the next website builder.

(Adding it always is better than having to remember when it's needed)